### DISCUSSION TITLE
Opening a large JSON file in GitHub website crashes instead of refusing to preview

### DISCUSSION THREAD

**Discussion by patarapolw** [community/community#72072] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The file in question is https://github.com/patarapolw/wiktionary-cjkdict/blob/c17433dde0ee6c51413e7253cac3606345faf63a/assets/cleaned/translingual.json

It appears to be slow to load at first, then it crashes.

![image](https://github.com/community/community/assets/21255931/12dc1b8d-bd82-4b6b-aeb6-540e2d47e075)

**Comment by Vijaykv5** [comment]
Yes that's true.
+1
